test/blackbox-tests/test-cases/alias-candidates.t/run.t

```diff
@@ -0,0 +1,8 @@
+Dune should suggest similar aliases when it cannot find one. 
```

Comment:
Can you turn that into a file cram test ? (`alias-candidate.t` with `cat > dune` and `cat > dune-project` in it)

Reply:
Sure, but why?

Reply:
that's fewer files, tests are more self-contained. I think we said we now prefer that for new tests.